.golangci.yml

```diff
@@ -241,21 +241,21 @@ linters:
     - govet
     # - errcheck
     - staticcheck
-    # - unused
+    - unused
     # - gosimple
     - ineffassign
     # - typecheck
     # Additional
     # - lll
     - godox
     #- gomnd
-    # - goconst
+    - goconst
     # - gocognit
     # - nestif
     # - gomodguard
     # - nakedret
     # - gci
-    # - misspell
+    - misspell
```

Comment:
this is worry. Can you tell me what is covered under misspell? Telecom has many acronym and will they cause misspell error?

Reply:
Overall, common English words. In general it should not car about acronyms

Reply:
what is the way to add exceptions? e.g. gNodeB, Amf, Smf, Upf ?

Reply:
No need to add exceptions because it checks commonly used English words. We have `AMF`, `UPF`, `UE `, `UL`, `GNodeB`, etc. in the code (as shown below) and there is no problem

```
// ConnectToPeer establishes SCTP connection with the AMF

gnbue.Log.Traceln("Sent UL Packet from UE to UPF")
 
// GnbUpTransport represents the User Plane transport of the GNodeB
```